### ISSUE TITLE
[Q] Not able to export ROC and prevision-recall curve?

### ISSUE THREAD

**Issue by roshan-k-patel** [wandb/client#2186]
Is exporting these graphs a paid feature only? I can export other graphs fine but there is no option to for these 
<img width="1400" alt="Screenshot 2021-05-10 at 8 04 57 PM" src="https://user-images.githubusercontent.com/48667731/117656898-7e472000-b1cb-11eb-9e82-50d8f45304ed.png">

**Comment by ariG23498** [comment]
Hey @roshan-k-patel 
I am not sure why `exporting` not an option in your case.

![image](https://user-images.githubusercontent.com/36856589/117693485-b6eef580-b1db-11eb-8518-a74bcb432c97.png)

I can indeed confirm that this is not a paid thing.
Could you send in the link of the run?

**Comment by roshan-k-patel** [comment]
hi @ariG23498, thanks for the response, [here](https://wandb.ai/rpatel9/deberta-adv-optimisation-high-epochs?workspace=user-rpatel9) is the workbook. The two runs with these charts runs are named: 
"colorful-glade-65 base mnli
classic-sound-63 large mnli"

**Comment by tyomhak** [comment]
Hey @roshan-k-patel , can you share how exactly you generated these graphs?

**Comment by roshan-k-patel** [comment]
@tyomhak, this is the script i used for those two runs with the roc curves. They are generated from the performance on the test set, might be part of the library i'm using (simpletransformers). My intention is to download the csv data and generate a nicer looking graph with pyplot or seaborn.

`# Python ≥3.5 is required
import sys
assert sys.version_info >= (3, 5)

# Scikit-Learn ≥0.20 is required
import sklearn
assert sklearn.__version__ >= "0.20"

# Common imports
import pandas as pd
import numpy as np
import os

import torch
import wandb
import sklearn.metrics
import os

df_train = pd.read_csv("train_df.csv")
df_test = pd.read_csv("test_df.csv")
df_eval = pd.read_csv("eval_df.csv")

os.environ["WANDB_API_KEY"]=""
os.environ["WANDB_ENTITY"]=""
os.environ["WANDB_PROJECT"]=""

# no internet run
os.environ["WANDB_MODE"] = "dryrun"

from simpletransformers.classification import ClassificationArgs, ClassificationModel

# Create a ClassificationModel
cuda_available = torch.cuda.is_available()

model = ClassificationModel(
    "deberta", "./deberta-base-mnli", num_labels=3, args={"reprocess_input_data": True, 
                                                             "overwrite_output_dir":True, 
                                                             "learning_rate":1.4138110972420529e-05, 
                                                             "num_train_epochs":5, 
                                                             "evaluate_during_training_verbose":True, 
                                                             "evaluate_during_training":True,
                                                             "manual_seed":42, 
                                                             "max_seq_length":512, 
                                                             'wandb_project': "deberta-adv-optimisation-high-epochs",
                                                             'params_classifier.dense.bias':0.0008231024558007287,
                                                             'params_classifier.dense.weight':6.694604159072914e-05,
                                                             'params_classifier.out_proj.bias':0.00021969884846106036,
                                                             'params_classifier.out_proj.weight':0.0009737859749708057,
                                                             'warmup_ratio':0.15875850752207474,
                                                             'weight_decay':0.1290002128443022,
                                                             'train_custom_parameters_only':False,
                                                             'evaluate_during_training_steps':100,
                                                             'use_early_stopping':True}, use_cuda=cuda_available)


# Train the model
model.train_model(df_train, eval_df=df_eval, accuracy=sklearn.metrics.accuracy_score, f1_macro = lambda truth, predictions: sklearn.metrics.f1_score(
    truth, predictions, average='macro'), f1_weighted = lambda truth, predictions: sklearn.metrics.f1_score(
    truth, predictions, average='weighted'))


# Evaluate the model
result, model_outputs, wrong_predictions = model.eval_model(df_test, acc=sklearn.metrics.accuracy_score, f1_macro = lambda truth, predictions: sklearn.metrics.f1_score(
    truth, predictions, average='macro'), f1_weighted = lambda truth, predictions: sklearn.metrics.f1_score(
    truth, predictions, average='weighted'))

wandb.finish()`

**Comment by ariG23498** [comment]
Hey @roshan-k-patel upon doing some digging I found out that `simpletranformers` use deprecated plotting commands. I will raise an issue with their team so that this can be taken care of. Thanks for the ticket!

**Comment by roshan-k-patel** [comment]
Thanks @ariG23498, Is there any possibility that you would be able to manually get me the roc and precision-recall curve csv data for both of those runs as i need them rather urgently? Thank you

**Comment by adrnswanberg** [comment]
Hi Roshan,

The data for these graphs is available to you in the 'Files' tab. Just pull up one of your runs, navigate to the 'Files' tab (the last one on the left-hand menu), and click into 'media/table'.

**Comment by roshan-k-patel** [comment]
Thank you!
